internal/driver/commands.go

```diff
@@ -129,115 +129,115 @@ var pprofCommands = commands{
 // reported generated by pprof.
 var pprofVariables = variables{
 	// Filename for file-based output formats, stdout by default.
-	"output": &variable{stringKind, "", "", helpText("Output filename for file-based outputs")},
+	"output": &variable{stringKind, []string{""}, "", helpText("Output filename for file-based outputs")},
```

Comment:
The fact that we need to change all these places to accept a list as the default value now makes me a bit uneasy.

Would the change be simpler if we'd say that the format for multiple tag filters is a comma separated string like `-tagfocus key1=value1,key1=value2`?